hr_timesheet_predefined_description/readme/USAGE.rst

```diff
@@ -0,0 +1,11 @@
+To use predefined descriptions in your timesheets:
```

Comment:
This file should contain the usage of the module, not the configuration. That's it how you select the predefined description in the timesheets, and they are reflected in the description.